changelog/683.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+ui: update documentation links in README and code comments
```

Comment:
For strictly internal documentation changes, we don't need a readme.

Reply:
Ok, i removed the commit.